<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customerEntry</class>
 <widget class="QMainWindow" name="customerEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>A1. Customer Entry</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="customersFrame1">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="customerIDLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Customer ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="customeNoLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Customer No:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputCustomerID">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputCustomerNo">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="customersFrame2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>601</width>
      <height>261</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="customerTypeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Customer Type: </string>
     </property>
    </widget>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Customer Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="addressOneLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Customer Address1:</string>
     </property>
    </widget>
    <widget class="QLabel" name="addressTwoLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Customer Address2:</string>
     </property>
    </widget>
    <widget class="QLabel" name="addressThreeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Customer Address3:</string>
     </property>
    </widget>
    <widget class="QLabel" name="addressFourLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Customer Address4:</string>
     </property>
    </widget>
    <widget class="QLabel" name="telephoneNoLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tel No:</string>
     </property>
    </widget>
    <widget class="QLabel" name="gstRegNoLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>GST Reg No:</string>
     </property>
    </widget>
    <widget class="QLabel" name="faxNoLabel">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>200</y>
       <width>55</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Fax No:</string>
     </property>
    </widget>
    <widget class="QLabel" name="accountNoLabel">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>230</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Account No:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="customerTypeDropBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputName">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>50</y>
       <width>421</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputAddrOne">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>421</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputAddrTwo">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>110</y>
       <width>421</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputAddrThree">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>421</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputAddrFour">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>170</y>
       <width>421</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputTelephone">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>200</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputGST">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>171</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputFax">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>200</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputAccountNo">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>230</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="tableFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>751</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="resetButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>70</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>110</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>150</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="printButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>190</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Print</string>
     </property>
    </widget>
    <widget class="QPushButton" name="saveButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>230</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="closeButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>270</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>601</width>
       <height>291</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QTableView" name="customerIDTable">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>581</width>
        <height>271</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>621</width>
       <height>41</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLineEdit" name="searchBar">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>341</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="findLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Find:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
